src/Files.App (Package)/Files.Package.wapproj

```diff
@@ -107,7 +107,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.3.230502000" IncludeAssets="build" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230518007-experimental1" IncludeAssets="build" />
```

Comment:
Do we need experimental? We want to release to stable in a few weeks and this will hold us back.

Reply:
The issue only repos in experimental for me, but I can also revert this.

Reply:
I can repro with the latest in `main`.

Reply:
Reverted.